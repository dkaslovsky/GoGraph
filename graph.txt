a b 1.5
a c 2
b c 3.3
c a 7
c d 1