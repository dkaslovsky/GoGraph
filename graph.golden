a b 1.5
a c 2
b c 3.3
c a 7
c d
c d 1.1